.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.183.0/containers/typescript-node
+{
+    "name": "Node.js & TypeScript",
+    "build": {
+        "dockerfile": "Dockerfile",
+        // Update 'VARIANT' to pick a Node version: 12, 14, 16
+        "args": {
+            "VARIANT": "12"
+        }
+    },
+    // Set *default* container specific settings.json values on container create.
+    "settings": {},
+    // Add the IDs of extensions you want installed when the container is created.
+    "extensions": [
+        "dbaeumer.vscode-eslint"
```

Comment:
are we using this extension? (or is this from a default template?)

conversely, we should add the prettier extension (though this might better be located in the workspace settings, so it's also used outside of the dev container?)

Reply:
Since VSCode generated the devcontainer.json file, i assume this is something that was present in my local dev setup. Removing the extension